InteractiveHtmlBom/web/ibom.js

```diff
@@ -659,7 +659,17 @@ function populateBomBody(placeholderColumn = null, placeHolderElements = null) {
           var valueSet = new Set();
           references.map(r => r[1]).forEach((id) => valueSet.add(pcbdata.bom.fields[id][field_index]));
           td = document.createElement("TD");
-          td.innerHTML = highlightFilter(Array.from(valueSet).join(", "));
+          var output = new Array();
+          const urlRegex = /^((file|https?):\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*))$/g;
+          for (let item of Array.from(valueSet)) {
+            const visible = highlightFilter(item);
+            if (item.match(urlRegex)) {
+              output.push('<a href="' + item + '" target="_blank">' + visible + '</a>');
```

Comment:
You can use backtick format strings (I avoided them in the past because edge didn't support a lot of stuff).
```
`<a href="${item}" target="_blank">${visible}</a>`
```

Reply:
Done.